Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -1410,7 +1403,12 @@ public void OnFocusExit(FocusEventData eventData)
         /// </summary>
         public void OnSpeechKeywordRecognized(SpeechEventData eventData)
         {
-            if (eventData.Command.Keyword == VoiceCommand && (!VoiceRequiresFocus || HasFocus) && IsEnabled)
+            if (!IsEnabled)
+            {
+                return;
+            }
```

Comment:
Curious here - in other locations in this PR, it looks like if clauses were joined to avoid multiple if->return but this one pulls it out - is this primarily to avoid a line count issue? If so we could just do something like:

```c#
if (eventData.Command.Keyword == VoiceCommand&&
    (!VoiceRequiresFocus || HasFocus) &&
    IsEnabled)
```

Reply:
Sort of, it was to make it more deliberate that we won't check or do anything if we aren't enabled.

I'll pull it out as a if (!isEnabled) {return} and then do the voice comamnd check. This will match the others

Reply:
I pulled the IsEnabled out for two reasons
1) I want it to be more deliberate and obvious to readers that we won't do anything unless enabled
2) It matches the rest of the file in that if we are disabled, return immediately